server/database/queries/getHistoryLogs/index.js

```diff
@@ -0,0 +1,12 @@
+const connection = require('../../config/connection');
+
+const getHistoryLogs = ({ patientId }) => {
+  const sql = {
+    text:
+      'SELECT history.price, history.payment, history.balance, history.description, appointments.appointment_date, appointments.patient_id FROM history INNER JOIN appointments ON appointments.id = history.appointment_id AND appointments.patient_id = $1 AND is_done = true;',
```

Comment:
There's no need for the patient id to send it twice in the response
```suggestion
      'SELECT history.price, history.payment, history.description, appointments.appointment_date FROM history INNER JOIN appointments ON appointments.id = history.appointment_id AND appointments.patient_id = $1 AND is_done = true;',
```

Reply:
done